src/backend/build-static.ts

```diff
@@ -111,7 +111,10 @@ export default async function buildStatic(): Promise<void> {
     await renderPage(page);
   }
 
-  await renderIndexPage(config.indexPageUri);
+  // Check if index page is enabled
+  if (config.indexPage.enabled) {
```

Comment:
do not forget to add docs for the new properties.

Reply:
Sure, but for now there are no docs about static building, it is experimental feature. I think, docs about it will appear later